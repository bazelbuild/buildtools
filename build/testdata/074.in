package()

load("foo", "x", "y")
load("foo", "x", "z")
load("bar", "baz")
