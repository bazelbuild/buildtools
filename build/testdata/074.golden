manifest = [
    # buildifier: table
    ("",                        "",      ""),
    ("ab",                      "bc",    "cd"),
    ("aaaaaaaaaaaaaaaaaaaa",    "b",     "c"),
]
